KUBERNETES_CSI_OWNERS_ALIASES

```diff
@@ -16,7 +16,7 @@ aliases:
   #
   # Reviewers are encouraged to set the "Busy" flag in their GitHub status
   # when they are temporarily unable to review PRs.
-  kubernetes-csi-reviewers:
+  reviewers:
```

Comment:
you can leave the `kubernetes-csi-` here. this is an alias.

Reply:
done